src/main/webapp/WEB-INF/views/cars/userAll.jsp

```diff
@@ -0,0 +1,42 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
```

Comment:
move this file to the drivers dir, because it relates to a driver. There is not user entity. You can name it `cars` for example